### DISCUSSION TITLE
How to save a credit card in Craft Commerce 3? Any help or tips would be appreciated

### DISCUSSION THREAD

**Discussion by DidoMarchet** [craftcms/commerce#3375] [Q&A]
Hi guys,
I have a question:

how can I save a credit card and use it again for the next payment in Craft Commerce 3?

I look some discussions about this question but didn't find an answer. 
If anyone could give me a simple example, that would be great!

Really thanks and kind regards,

Davide

**Comment by jelmer-scoremedia** [comment]
Hi Davide,

I don't think saving creditcards is something you want to do on your end. See https://www.lawpay.com/about/blog/storing-credit-card-information/

I'd suggest using a (supported) payment gateway like Stripe, Mollie, PayNL etc.. By integrating with a reputable payment gateway, you not only enhance the security of your customers' sensitive data but also streamline your payment processes, offering a smoother and more trustworthy experience for your users.

**Reply by DidoMarchet** [reply]
Hi thanks, 
I'm using Stripe as gateway but I didn't find any example 
or documentation how to do that with the craft commerce Stripe Checkout plugin.

I don't want to manage user's card but I want, as you said, offer a smooth experience.

If you have experience or an example would be appreciated.

And really thanks again,
kind regards,

Davide
